### PR TITLE
Update active-user-in-usage-reports.md

### PR DESCRIPTION
**Pull request by svemu**
Update Teams Active user definition to include other activities

### CODE REVIEW

**Comment by mypil** [comment]
@Sirkkuw - can you please assist to check, approve and merge the content for this PR? 

Thank you.

**Comment by mypil** [comment]
@Sirkkuw - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR?

Thank you.

**Comment by dianef77** [comment]
Please ask a copy editor to review

**Comment by mypil** [comment]
@dianef77 - already assigned @TokyoScarab  to do the copy editor review. Thanks.

**Comment by Sirkkuw** [comment]
@mypil done

**Comment by TokyoScarab** [comment]
There are a couple missing commas and a missing space. Here it is with the changes
|Microsoft Teams  <br/> |Any user who has participated in chat messages, private chat messages, calls, meetings or other activity. Other activity is defined as the number of other team activities by the user, some of which include, and not limited to: liking messages, apps, working on files, searching, and following or favoriting teams and channels.  <br/> ||

File: Office365-Admin/usage-analytics/active-user-in-usage-reports.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ An active user of Office products for [Microsoft 365 usage analytics](usage-anal
...
 |Office  <br/> |Any user who has activated their Office 365 Pro Plus, Visio Pro or Project Pro subscription on at least one device.  <br/> ||
 |Office 365 Groups  <br/> |Any group member that has mailbox activity (if a message has been sent to the group)  <br/> |This definition will be enhanced with group site file activity and Yammer group activity (file activity on group site and message posted to Yammer group associated with the group.) This data is currently not available in the Office 365 Adoption content pack  <br/> |
-|Microsoft Teams  <br/> |Any user who has participated in chat messages, private chat messages, calls, or meetings.  <br/> ||
+|Microsoft Teams  <br/> |Any user who has participated in chat messages, private chat messages, calls,meetings or other activity. Other activity is defined as the number of other team activities by the user some of which include, and not limited to: liking messages, apps, working on files, searching, following teams and channel and favoriting them.  <br/> ||
```
Comment (TokyoScarab) [inline Office365-Admin/usage-analytics/active-user-in-usage-reports.md]: |Microsoft Teams
|Any user who has participated in chat messages, private chat messages, calls, meetings or other activity. Other activity is defined as the number of other team activities by the user, some of which include, and not limited to: liking messages, apps, working on files, searching, and following or favoriting teams and channels.
||

**Pull request merged**